### PR TITLE
Add Cluster Metadata in Status Page

### PR DESCRIPTION
**Pull request by eolivelli**
- read Auditor and replication status from ZK
- clean up the System status page

### CODE REVIEW

File: web/src/main/bvmui/src/router.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ const router = new Router({
             name: 'cache',
             component: Cache,
```
Comment (mino181295) [inline web/src/main/bvmui/src/router.js]: Can you alsto rename the .vue file?

File: web/src/main/java/org/bookkeepervisualmanager/api/resources/SystemStatusResource.java [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        return new ClusterWideConfiguration(clusterWideConfiguration.getAuditor(),
+                clusterWideConfiguration.isAutorecoveryEnabled(),
+                clusterWideConfiguration.getLostBookieRecoveryDelay(),
+                clusterWideConfiguration.getLayoutFormatVersion(),
+                clusterWideConfiguration.getLayoutManagerFactoryClass(),
+                clusterWideConfiguration.getLayoutManagerVersion(),
+                refreshWorkerStatus.getBookkkeeperClientConfiguration());
+    }
+
+    public static final class ClusterWideConfiguration {
```
Comment (mino181295) [inline web/src/main/java/org/bookkeepervisualmanager/api/resources/SystemStatusResource.java]: Maybe it's better in a separate class this time?

**Comment by eolivelli** [comment]
@mino181295 I have addressed all of your comments and fixed spotbugs.

File: web/src/main/java/org/bookkeepervisualmanager/api/resources/SystemStatusResource.java (hunk 2 above)
Comment (eolivelli) [reply web/src/main/java/org/bookkeepervisualmanager/api/resources/SystemStatusResource.java]: this class is not used. I have dropped it.
I have moved LedgerBean to a separate class

File: web/src/main/bvmui/src/router.js (hunk 1 above)
Comment (eolivelli) [reply web/src/main/bvmui/src/router.js]: done

**Pull request merged**